yarn.lock

```diff
@@ -8405,7 +8405,7 @@ watchpack@^1.6.1:
     graceful-fs "^4.1.2"
     neo-async "^2.5.0"
 
-wavesurfer.js@^3.3.3:
+wavesurfer.js@3.3.3:
```

Comment:
This is a manual change?

Reply:
Nope. Forgot to run`yarn install` when changed `^3.3.3` to `3.3.3`.